### ISSUE TITLE
in canvas cards, triple-backtick text is drawn in a proportional font, not mono

### ISSUE THREAD

**Issue by drewp** [dracula/obsidian#31]
![image](https://user-images.githubusercontent.com/104003/227406616-9ed36c3f-410f-4219-9e0d-57d5f931f603.png)

On this canvas card, I expected the cyan words to be in a monospace font, like they are in a non-canvas note.

Here's a non-canvas note:
![image](https://user-images.githubusercontent.com/104003/227406957-faba8082-6867-4be0-ad58-cf4e672cfb3b.png)

**Comment by Chrismettal** [comment]
I think you are in the wrong repo with this one.

This is what a canvas card looks like in the official Dracula theme, where this issue is posted:
![image](https://user-images.githubusercontent.com/39636046/227738916-7fb8eecb-5aec-445e-8eaa-557b5f554d32.png)


This is with the [unofficial Dracula theme by jarodise](https://github.com/jarodise/Dracula-for-Obsidian.md):
![image](https://user-images.githubusercontent.com/39636046/227738896-e32f6397-051f-4e0d-8b60-453e72d9b67c.png)

So while your issue strictly speaking has nothing to do with this repo, cards in the official theme DO look wrong in their own way by using the wrong colors and not using correct background colors.

**Comment by Chrismettal** [comment]
I have updated the theme to not override the background color so it allows setting the card color via the canvas color chooser. I also added the Dracula colors to said color chooser.

So while you are talking about a different theme entirely, you helped me identify issues with this new cards feature that I wasn't aware of. Thank you!

![image](https://user-images.githubusercontent.com/39636046/227740494-7cfc92e4-0111-4b4b-b1b6-d92f36e3ec57.png)
